packages/xy-chart/src/utils/chartUtils.js

```diff
@@ -77,6 +77,27 @@ export function propOrFallback(props, propName, fallback) {
   return props && isDefined(props[propName]) ? props[propName] : fallback;
 }
 
+export function scaleInvert(scale, value) {
+  if (!scale.invert) {
+    const leftEdges = scale.range();
+    let i = 0;
+    const width = scale(scale.domain()[1]) - scale(scale.domain()[0]);
```

Comment:
could this be replaced with [`scale.step`](https://github.com/d3/d3-scale#band_step) since this should only be hit for band scales?